src/src/rx-serial/SerialMavlink.cpp

```diff
@@ -49,14 +50,16 @@ void SerialMavlink::sendQueuedData(uint32_t maxBytesToSend)
 
 SerialMavlink::SerialMavlink(Stream &out, Stream &in):
     SerialIO(&out, &in),
-    // 255 is typically used by the GCS, for RC override to work in ArduPilot `SYSID_MYGCS` must be set to this value (255 is the default)
-    this_system_id(255),
-    // Strictly this is not a valid source component ID
-    this_component_id(MAV_COMPONENT::MAV_COMP_ID_ALL),
-    // Assume vehicle system ID is 1, ArduPilot's `SYSID_THISMAV` parameter. (1 is the default)
-    target_system_id(1),
-    // Send to AutoPilot component
-    target_component_id(MAV_COMPONENT::MAV_COMP_ID_AUTOPILOT1)
+    
+    //system ID of the device component sending command to FC, can be set using lua options, 0 is the default value for initialized storage, threat it as 255 which is commonly used as GCS SysID
```

Comment:
Is this comment supposed to say `treat it as 255` instead of `threat it as 255`?
"Treat" seems to fit the sentence. "Threat" not so much...